docs/howtos/Release_Process_with_PR_Rebase.md

```diff
@@ -0,0 +1,35 @@
+# Release Process with PR Rebase
```

Comment:
I think it'd be good to mention what's the purpose of this guide a bit more clearly. Also, let's use the language of branches rather than that of releases, since releases are rather internal, and it may be confusing especially for contributors.

I imagine that the introductory section on lines 2-6 could therefore say rather something like:  

> On some occasions, a pull request may need to be rebased from the `develop` branch to the latest release branch `release-v*` (for example, `release-v0.16.x` if we currently work on version 0.16) or vice versa. This guide shows how to do so while  keeping the commit history clean.
> 
> This process is demonstrated on rebasing a pull request originally targeting `develop` branch to the latest release branch `release-v*`. If you need to rebase your pull request in the other direction, from `release-v*` to `develop`, you can use the very same steps - just switch branches names in the guide below.